include/IVideoDriver.h

```diff
@@ -171,7 +172,7 @@ namespace video
                                                            asset::E_FORMAT format = asset::EF_B8G8R8A8_UNORM, const bool& fixedSampleLocations = false) {return nullptr;}
 
         //! A.
-        virtual ITextureBufferObject* addTextureBufferObject(IGPUBuffer* buf, const ITextureBufferObject::E_TEXURE_BUFFER_OBJECT_FORMAT& format = ITextureBufferObject::ETBOF_RGBA8,
+        virtual IGPUBufferView* addTextureBufferObject(IGPUBuffer* buf, asset::E_FORMAT format = asset::EF_R8G8B8A8_UNORM,
```

Comment:
do we have a query for supported formats for TBO?

Also call it `addBufferView`

Reply:
nah, but there should be such. I'll make later

Reply:
+1

Reply:
resolved